src/controller.cpp

```diff
@@ -207,17 +194,18 @@ int Controller::run(const CliArgsParser& args)
 	std::string cachefilepath = cfg.get_configvalue("cache-file");
 	if (cachefilepath.length() > 0 && !args.cache_file().has_value()) {
 		configpaths.set_cache_file(cachefilepath);
-		fslock = std::unique_ptr<FsLock>(new FsLock());
-		pid_t pid;
-		if (!fslock->try_lock(configpaths.lock_file(), pid)) {
-			std::cout << strprintf::fmt(
-					_("Error: an instance of %s is "
-						"already running (PID: %u)"),
-					PROGRAM_NAME,
-					pid)
-				<< std::endl;
-			return EXIT_FAILURE;
-		}
+	}
+
+	fslock = std::unique_ptr<FsLock>(new FsLock());
```

Comment:
Since we're initializing `fslock` in just one place now, I think we can replace `std::unique_ptr<FsLock>` with just `FsLock`, and let the constructor run the initialization.

Reply:
Good idea, done